content/applications/websites/ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,58 @@
-==================================
-How to customize my catalog page 
-==================================
+=================
+Customize catalog
+=================
 
-Product Catalog
+Product catalog
 ===============
 
-All your published items show up in your catalog page (or *Shop* page).
+All your published items show up in your catalog (or *Shop*) page.
 
-.. image:: ./media/shop.png
+.. image:: catalog/product-catalog.png
    :align: center
+   :alt: ecommerce product catalog page
 
-Most options are available in the *Customize* menu: display attributes,
-website categories, etc.
+Most modification options and features are available in the *Customize* menu.
 
-.. image:: ./media/shop_customize.png
+.. image:: catalog/catalog-customize-menu.png
    :align: center
+   :alt: ecommerce customize menu on product catalog page
 
 Highlight a product
 ===================
 
-Boost the visibility of your star/promoted products: push them to top, make them
-bigger, add a ribbon that you can edit (Sale, New, etc.). Open the Shop page, switch
-to Edit mode and click any item to start customizing the grid.
+Boost the visibility of your star (or promoted) products in a number of different ways. Push them
+to top, make them bigger, add an editable ribbon to attract attention (*Sale*, *New*, etc.).
 
-See how to do it: https://www.odoo.com/openerp_website/static/src/video/e-commerce/editing.mp4
+Simply open the *Shop* page, switch to *Edit* mode, and click any item to start customizing.
 
-Quick add to cart
-=================
+.. tip::
+   See how to do it `here <https://www.odoo.com/openerp_website/static/src/video/e-commerce/editing
+   .mp4>`_.
+   (**NOTE**: *Footage is from Odoo v13, but the methodology still applies.*)
+
+Quick 'Add to Cart'
+===================
+
+If customers tend to buy a lot of items at once, you can make their shopping experience even
+*faster* by enabling an "Add to Cart" button directly on products found on your catalog page.
```

Comment:
why *faster*?

Reply:
I did my best to explain how this could expedite the purchasing process. Please review my explanation and reach out to me if there needs to be more elaboration on the subject. Thanks!